### PR TITLE
test(acceptance): Try #2 to fix flakey onboarding test for django 1.10

### PR DESCRIPTION
**Pull request by billyvg**
Try #2, use a timed retry policy

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,6 +32,7 @@ env:
     - PIP_QUIET=1
     - SENTRY_LIGHT_BUILD=1
     - SENTRY_SKIP_BACKEND_VALIDATION=1
+    - SENTRY_DSN=https://0e3f4096f0ec40ad871edc7b9e0c489f@sentry.io/282218
```
Comment (dashed) [inline .travis.yml]: I'm curious to see how well this works.

**Comment by dashed** [comment]
@billyvg did the tracing instrumentation help at all for this PR?

**Review by dashed — APPROVED** [review]
Looks like it's not flakey to me 👍

**Comment by billyvg** [comment]
@dashed no I couldn’t get server side tracing working :/

**Pull request merged**